feature/experimental/isis/ate_tests/internal/session/session.go

```diff
@@ -105,6 +105,7 @@ func ProtocolPath() *networkinstance.NetworkInstance_ProtocolPath {
 func addISISOC(dev *oc.Root, areaAddress, sysID, ifaceName string) {
 	inst := dev.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
 	prot := inst.GetOrCreateProtocol(PTISIS, ISISName)
+	prot.Enabled = ygot.Bool(true)
```

Comment:
Curious - why add this?  I don't think enabling/disabling individual routing protocols is widely supported.

Reply:
Looks like I'm wrong - IOS-like NOS's seem to support a 'shutdown' command under protocols.  Still curious about the utility of this leaf for this test.

Reply:
@aredmon8551 does this [comment](https://github.com/openconfig/featureprofiles/pull/745#issuecomment-1314620480) answer your question? We are not enforcing it. We want to keep things moving while seeking clarification at the source (the OC)

Reply:
Thanks!

Somewhat answers the question - though I'm missing the motivation for the addition - is this change (plus the addition of a deviation) needed to allow support for all NOS's?  Or is this for consistency?  I'm not aware of this being a mandatory leaf/config node in any implementation.  Perhaps I'm missing something.

Reply:
The referenced comment basically suggests to push things forward via 2 tracks in parallel: 

1. Align with OC. The OC doesn't set default value, so we need to set the value. That's what this PR is about. Meanwhile, let's create a deviation for other vendors that does not support set it explicitly. 
3. I'm going to file a PR to OC suggesting set the default value true. 
4. If 2 merged, let's converge on what test behaviors we want. In that case, probably we will remove the requirement of explicitly set it true.

Reply:
Ack, makes sense.  Thanks!